Source/DafnyCore/Resolver/PreType/PreTypeConstraints.cs

```diff
@@ -583,6 +583,8 @@ public enum CommonConfirmationBag {
       preType = preType.Normalize();
```

Comment:
Can you add a comment to `ConfirmConstraint`, or update its signature (parameter names) to make it clear what it does? I don't understand the meaning of the parameters or return value by looking at just its signature and comment.

Reply:
I can't, as I don't understand the code at a level high enough to do that. I don't have a mental model of what ConfirmConstraint does, nor about what code context it was called with. I'll leave it for @RustanLeino  to comment on ti.